tests/test_cli_e2e.py

```diff
@@ -180,14 +180,20 @@ def verify_files(files, base=""):
             assert os.path.getsize(full_name) == value
 
 
-def run_step(step):
+def run_step(step, catalog):
     """Run an end-to-end test step with a command and expected output."""
     result = subprocess.run(  # noqa: S603
         step["command"],
         shell=False,
         capture_output=True,
         check=True,
         encoding="utf-8",
+        env={
```

Comment:
[F] This is my attempt to ensure the database is isolated to this set of e2e tests